composer.json

```diff
@@ -34,7 +35,7 @@
         "sonata-project/user-bundle": "For user timezone detection"
     },
     "conflict": {
-        "sonata-project/user-bundle": "<2.0 || >=4.0"
+        "sonata-project/user-bundle": "<2.0 || >=5.0"
```

Comment:
IMO we should use conflict section when we detect a conflict with a concrete version of a bundle, not to be sure it does not break in the future, so I agree with you. Just added it to test what @Soullivaneuh said.

That being said, we need to get an agreement here, because I can't follow those 2 messages at the same time: keep the upper limit, and not keep it.